src/PaymentGateways/Gateways/Stripe/StripePaymentElementGateway/stripePaymentElementGateway.tsx

```diff
@@ -119,7 +121,13 @@ const stripePaymentElementGateway: StripeGateway = {
         const {error: submitError} = await this.elements.submit();
 
         if (submitError) {
-            throw new Error(submitError);
+            throw new Error(
+                sprintf(
+                    __('Invalid Payment Data. Error Details: %s (code: %s)', 'give'),
+                    submitError.message,
+                    submitError.code
```

Comment:
Are we sure the submitError object always has these properties?

Reply:
Resolved in 2a194133519344890d04d84bfd9f4d4a791dde7e